lib/modules/manager/gitlabci/utils.ts

```diff
@@ -10,3 +14,21 @@ export function replaceReferenceTags(content: string): string {
   const res = content.replace(re, '');
   return res;
 }
+
+const depProxyRe = regEx(
+  `(?:\\$\\{?CI_DEPENDENCY_PROXY_(?:DIRECT_)?GROUP_IMAGE_PREFIX\\}?\\/)?(?<depName>.+)`
+);
+
+/**
+ * Get image dependencies respecting Gitlab Dependency Proxy
+ * @param imageName as used in .gitlab-ci.yml file
+ * @return package dependency for the image
+ */
+export function getGitlabDep(imageName: string): PackageDependency {
+  const match = depProxyRe.exec(imageName);
+  if (match?.groups) {
+    return getDep(match.groups.depName);
```

Comment:
```suggestion
    return { ...getDep(match.groups.depName), replaceString: imageName ];
```

I would prefer to have the full name for replacement, so it's more unique for auto-replace.

@rarkins WDYT?

Reply:
@rarkins ☝️

Reply:
Yes, that would be better